### PR TITLE
feat: support to use path parameter with plugin's control api

### PR DESCRIPTION
**Pull request by The-White-Lion**
### What this PR does / why we need it:
This PR resolves issue #5882 by adding the following configuration to the `config.yaml` to enable path parameter support for the control api in the plugins
```yaml
apisix:
    router:
        control: 'radixtree_uri_with_parameter'
```

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Could you put the router configuration under the `control` section, like
```
enable_control: true
control:
  # <- here
  #  ip: 127.0.0.1
  #  port: 9090
```

Thanks!

**Comment by The-White-Lion** [comment]
> Could you put the router configuration under the `control` section, like
> 
> ```
> enable_control: true
> control:
>   # <- here
>   #  ip: 127.0.0.1
>   #  port: 9090
> ```
> 
> Thanks!

yes， I followed you suggestion to modify the configuration like this

```
enable_control: true
control:
  #  router: 'radixtree_uri_with_parameter'
  #  ip: 127.0.0.1
  #  port: 9090
```

**Review by spacewander — COMMENTED** [review]
Could you update https://github.com/apache/apisix/blob/master/docs/en/latest/control-api.md?

File: apisix/control/router.lua [hunk 1]
```diff
@@ -95,6 +96,14 @@ do
 function fetch_control_api_router()
     core.table.clear(routes)
 
+    local with_parameter = false
+    local conf = core.config.local_conf()
+    if conf.apisix.enable_control and conf.apisix.control then
```
[the remark this answers is not in the document]
Comment (The-White-Lion) [reply apisix/control/router.lua]: > Could you update https://github.com/apache/apisix/blob/master/docs/en/latest/control-api.md?
This is my oversight that forgot to update the related documention.

Now, I update https://github.com/apache/apisix/blob/master/docs/en/latest/control-api.md and https://github.com/apache/apisix/blob/master/docs/zh/latest/control-api.md.


> Could you move this block to the end of the function? Also, we can save `conf.apisix.enable_control and ` as control API must be enabled if we hit this.

I moved this block to the end of the function.
Thanks for your patient guidance！

File: docs/en/latest/control-api.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ apisix:
     port: 9090
 ```
 
+The control API server does not support parameter matching by default, if you want to enable parameter matching
+you can add `router: 'radixtree_uri_with_parameter'` to the `control` section.
```
Comment (spacewander) [inline docs/en/latest/control-api.md:42]: Let's add a link to https://github.com/apache/apisix/blob/master/docs/en/latest/router-radixtree.md#5-parameter-match

**Review by spacewander — APPROVED** [review]

File: docs/en/latest/control-api.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ apisix:
     port: 9090
 ```
 
+The control API server does not support parameter matching by default, if you want to enable parameter matching
```
Comment (spacewander) [inline docs/en/latest/control-api.md:41]: We should notify that the parameter matching only work with the API added by plugin, not the builtin one from APISIX.

Comment (The-White-Lion) [reply docs/en/latest/control-api.md]: > 
I update the documention about control-api.md and indicate the config will efficetive with plugin's control API

File: docs/en/latest/control-api.md [hunk 4]
```diff
@@ -38,6 +38,9 @@ apisix:
     port: 9090
 ```
 
+The control API server does not support parameter matching in plugin's control API by default, if you want to enable parameter matching
```
Comment (spacewander) [inline docs/en/latest/control-api.md:41]:
```suggestion
The control API server does not support parameter matching by default, if you want to enable parameter matching in plugin's control API
```

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**